### DISCUSSION TITLE
请教用homebrew装，是有多少依赖呀

### DISCUSSION THREAD

**Discussion by duchengyao** [yutto-dev/yutto#260] [Q&A] [answered]
我感觉有成百上千依赖了吧，都要傻眼了哈哈哈哈哈，请问有没有在mac里面更方便的安装方法

**Comment by lxgdaove1503** [comment]
没那么多

**Comment by SigureMo** [comment, marked as answer]
https://github.com/SigureMo/homebrew-tap/blob/main/Formula/yutto.rb#L15-L96

没啥依赖啊，也就 FFmpeg 的依赖多一些吧

**Reply by duchengyao** [reply]
原来是ffmpeg的依赖多，谢谢解答^_^
